doc/DEPLOYMENT.md

```diff
@@ -372,7 +373,34 @@ This module uses fastcgi and shibboleth scripts to provide similar integration a
 
  1. Starting from Ubuntu Bionic or NGINX 1.11 we can dynamic modules
 
-    **Write down instructions for this.**
+        # as root
+        cd /usr/src
+
+        apt-get install build-essential devscripts
+        # edit /etc/apt/sources.list if necessary
+        apt-get source nginx
+        apt-get build-dep nginx
+
+        git clone https://github.com/nginx-shib/nginx-http-shibboleth.git
+        git clone https://github.com/openresty/headers-more-nginx-module
+
+        pushd nginx-1.*/
+        # check current nginx flags with `nginx -V` and append
+        # them after the configure command
+        ./configure \
+          --add-dynamic-module=../nginx-http-shibboleth \
+          --add-dynamic-module=../headers-more-nginx-module \
+          FLAGS_HERE
+        make modules
+        chmod 644 \
+           objs/ngx_http_shibboleth_module.so \
+           objs/ngx_http_headers_more_filter_module.so
+        cp objs/ngx_http_shibboleth_module.so \
+           objs/ngx_http_headers_more_filter_module.so \
+           /usr/lib/nginx/modules/
```

Comment:
Can this be `/usr/local/lib/nginx/modules/`?

p.s. in Aalto, could CS IT or Aalto package prebuild modules as a debian package and install that with the nginx on relevant servers?

Reply:
Module path changed to `/usr/local/lib/nginx/modules`. Technically the module could be prebuilt as a debian package but I'd say that setting up shibboleth is such a rare case that it's not worth the effort